app/AppKernel.php

```diff
@@ -70,6 +70,10 @@ public function registerBundles()
             new AppBundle\AppBundle(),
         ];
 
+        // Symfony 3.3 moved server:* commands to another bundle
```

Comment:
the sandbox need not be generic. lets require that web server bundle and simply instantiate it